modelon/impact/client/entities/workspace.py

```diff
@@ -545,3 +629,117 @@ class object.
             self._model_exe_sal,
             self._exp_sal,
         )
+
+    def _refresh(self):
+        self._workspace_definition = WorkspaceDefinition.from_dict(
+            self._workspace_sal.workspace_get(self._workspace_id)['definition']
+        )
+
+    def get_projects(self):
+        """Return the list of projects for a workspace.
+
+        Returns:
+
+            projects --
+                A list of modelon.impact.client.entities.project.Project
+                class object.
+
+        Example::
+
+            projects = workspace.get_projects()
+        """
+        resp = self._workspace_sal.projects_get(self._workspace_id)
+        projects = [
+            Project(
+                item["id"],
+                ProjectDefinition.from_dict(item['definition']),
+                self._project_sal,
+            )
+            for item in resp["data"]["items"]
+        ]
+        return projects
+
+    def get_dependencies(self, include_disabled: bool = True):
```

Comment:
Skip `include_disabled` arg, the user can handle this themself if needed. If we find that this is an important feature we should probably implement it into the REST end-point so the client can be stupid.